code/cavity2d/CAVITY_2D_STAGGERED.F90

```diff
@@ -0,0 +1,507 @@
+!==========================2D CAVITY FLOW=========================================  

```

Comment:
This file uses DOS format (CRLF line termination), please change to UNIX format (only LF).

Reply:
Sorry, I never used unix format before. I will try to learn UNIX format.

Reply:
Your other file uses Unix line termination.  A decent editor, e.g., vim, allows you to elegantly control it.